conf.py

```diff
@@ -231,3 +232,9 @@
 # Output file base name for HTML help builder.
 htmlhelp_basename = 'PythonCoursewareProjectdoc'
 
+# for cusotm jinja2 filter
+import jinja2
+from runestone.utility import extractText, extractTextII
+
+jinja2.filters.FILTERS['extractText'] = extractText
```

Comment:
Ohhh.... I think this stuff could all go in the setup function

Reply:
With the above change, the Travis CI is running without error, and all the Utility components stuff is in that component. I guess this PR can be closed.